lgc/builder/BuilderReplayer.cpp

```diff
@@ -432,12 +432,12 @@ Value *BuilderReplayer::processCall(unsigned opcode, CallInst *call) {
 
   // Replayer implementations of DescBuilder methods
   case BuilderRecorder::Opcode::LoadBufferDesc: {
+    assert(call->getType()->getNonOpaquePointerElementType() == m_builder->getInt8Ty());
```

Comment:
Can you write this assert in a way so that it works with opaque pointers as well? (e.g. with isOpaqueOrPointeeTypeMatches)

Reply:
Yes, I will update this assert, but first this PR need to be merged : https://github.com/GPUOpen-Drivers/llpc/pull/1876